instructure/canvas-ios
# Message instructor from grades part 1

refs: MBL-11246
affects: Parent
release note: none

Test: 
1. make sure reply button does not show on grades if parentInbox flag is disabled
2. make sure compose view is shown modally if button is clicked

There will be a subsequent ticket for parts of MBL-11246 and the compose message details.
